frontend/src/components/TopNavigation.tsx

```diff
@@ -49,8 +50,10 @@ const TopNavigation: FunctionComponent<Props> = ({ hideControls }) => {
   const { user, logout } = useAuth0()
   const { profile } = useContext(UserProfileContext)
 
-  const filteredNavLinks = NAV_LINKS.filter(
-    (link) => link.adminOnly === profile?.isAdmin,
+  const userIsAdmin = profile?.isAdmin
+
+  const filteredNavLinks = NAV_LINKS.filter((link) =>
+    link.adminOnly ? userIsAdmin : true,
```

Comment:
Previous/incorrect logic:
- display admin-only links to admins
- display non-admin-only links to non-admins

Updated/correct logic:
- display all links to admins
- hide admin-only links from non-admins